opendevin/sandbox/ssh_box.py

```diff
@@ -371,6 +402,12 @@ def close(self):
                 ssh_box.kill_background(bg_cmd.id)
                 logger.info('Background process killed')
                 continue
+            if user_input.startswith('py:'):
```

Comment:
Maybe we should have a `JupyterAgent` or something to demonstrate this functionality?

Reply:
For this PR, I'm just trying to get the sandbox component ready :) I'll submit follow-up PRs to update `CodeActAgent` using the sandbox with instructions on how to use the fine-tuned open-source models (CodeAgentAgent-Mistral-7b) and can be just run on a laptop!